codefresh.yml

```diff
@@ -37,3 +38,10 @@ steps:
       - 'git config --global --add url."git@github.com:".insteadOf "https://github.com/"'
       - "make testacc"
     stage: "test"
+
+  notify:
```

Comment:
I would use the official plugin for this https://codefresh.io/steps/step/slack-notifier

Reply:
Done. I had to use directly docker image of plugin instead of a step way described in doc - there is a bug/feature which prevents to use helpful variables: `SLACK_USER_NAME` and `SLACK_EMOJI_ICON` (plugin validation blocks them)